src/style.rs

```diff
@@ -308,6 +320,22 @@ impl Style {
         self
     }
 
+    /// Changes the underline color. The text must be underlined with a modifier for this to work.
+    ///
+    /// ## Examples
+    ///
+    /// ```rust
+    /// # use ratatui::style::{Color, Modifier, Style};
+    /// let style = Style::default().underline_color(Color::Blue).add_modifier(Modifier::UNDERLINED);
+    /// let diff = Style::default().underline_color(Color::Red).add_modifier(Modifier::UNDERLINED);
+    /// assert_eq!(style.patch(diff), Style::default().underline_color(Color::Red).add_modifier(Modifier::UNDERLINED));
+    /// ```
+    #[cfg(feature = "crossterm")]
```

Comment:
Can you please link to the spec here (and note that the reason the feature is only available in the crossterm feature is because it's not yet implemented on the other backends).

Reply:
As it's non-standard I don't think there's an official document for it. I can either link to [Wikipedia](https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_(Select_Graphic_Rendition)_parameters) or [Kitty's documentation](https://sw.kovidgoyal.net/kitty/underlines/)

Reply:
Either is fine - just something to help those that come behind.